Update start-layout-xml-desktop.md (#4525)
MicrosoftDocs/windows-itpro-docs: windows/configuration/start-layout-xml-desktop.md

>- Do not add multiple rows of comments.

The following table lists the supported elements and attributes for the LayoutModification.xml file.
> [!NOTE]

> > RequiredStartGroupsCollection and AppendGroup syntax only applies to the special case of the Import-StartLayout method used for building and deploying Windows images.

Reviewer: ```suggestion
> RequiredStartGroupsCollection and AppendGroup syntax only applies when the Import-StartLayout method is used for building and deploying Windows images.
```
